client/src/components/App.jsx

```diff
@@ -11,10 +11,12 @@ class App extends React.Component {
       allReviews: [],
       currentReviews: [],
       currentSearchTerm: null,
+      allAverageRatings: {},
     }
     this.searchSubmit = this.searchSubmit.bind(this);
     this.searchBarTextChange = this.searchBarTextChange.bind(this);
     this.calculateAverageRating = this.calculateAverageRating.bind(this);
+    this.storeAllAverageRatings = this.storeAllAverageRatings.bind(this);
   }
   
   componentDidMount() {
```

Comment:
Would be a bit nicer if this function was written without storing context in a constant